src/System.Management.Automation/engine/GetCommandCommand.cs

```diff
@@ -599,9 +599,7 @@ private void AccumulateMatchingCmdlets()
         {
```

Comment:
@iSazonov, your last commit had 4 failures in `PowerShell-CI-macos`
Verify approved aliases list.All approved Cmdlets present (no new Cmdlets added, no Cmdlets removed)
```powershell
Object reference not set to an instance of an object.
at <ScriptBlock>, /Users/vsts/agent/2.154.1/work/1/s/test/powershell/engine/Basic/DefaultCommands.Tests.ps1: line 533
533:         $currentCmdletList = (Get-Command -CommandType Cmdlet | Where-Object { $moduleList -contains $_.Source }).Name

```
Verify approved aliases list.All present Cmdlets should have the correct ConfirmImpact
```powershell
Object reference not set to an instance of an object.
at <ScriptBlock>, /Users/vsts/agent/2.154.1/work/1/s/test/powershell/engine/Basic/DefaultCommands.Tests.ps1: line 548
548:         $currentCmdletList = Get-Command -CommandType Cmdlet |

```
TabCompletion.NativeCommand.Error occurred in Context block
```powershell
Object reference not set to an instance of an object.
At /Users/vsts/agent/2.154.1/work/1/s/test/powershell/Host/TabCompletion/TabCompletion.Tests.ps1:157 char:31
```
Export-Alias.Should create a file with the list of aliases that match the expected list
```powershell
Object reference not set to an instance of an object.
at <ScriptBlock>, /Users/vsts/agent/2.154.1/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/Export-Alias.Tests.ps1: line 155
155: 		$expected = Get-Command -CommandType Alias

```